pkgs/applications/networking/browsers/w3m/default.nix

```diff
@@ -12,8 +12,6 @@ assert graphicsSupport -> imlib2 != null;
 assert x11Support -> graphicsSupport && libX11 != null;
 assert mouseSupport -> gpm-ncurses != null;
```

Comment:
I forgot to mention: those asserts can now also be deleted

Reply:
Removed!